web/api/sector/sector.go

```diff
@@ -96,7 +98,7 @@ func (c *cfg) getSectors(w http.ResponseWriter, r *http.Request) {
 	// TODO get sector info from chain and from database, then fold them together
 	// and return the result.
 	type sector struct {
-		MinerID        int64 `db:"miner_id"`
+		MinerID        int64 `db:"miner_id" json:"-"`
```

Comment:
Why do we need this json key?

Reply:
Because the frontend part no longer requires MinerID, but the Go part still does, it is hidden.

Reply:
Okay. We should have been okay even if we sent this key to frontend.

Reply:
Should I remove this json tag?

Reply:
Let's remove it for now. We might end up needing that ID later.